gui/templates/storage/volume_detach.html

```diff
@@ -22,10 +22,22 @@
         </td>
     </tr>
     {% endif %}
-    {% if services %}
+    {% if attachments %}
     <tr>
         <td colspan="2">
-        <p>{% trans "ATTENTION - Currently the following services depend on this volume: " %}<strong>{{ services.keys|join:", " }}</strong></p>
+            {% trans "ATTENTION - Currently the following services depend on this volume: " %}
```

Comment:
```suggestion
            {% trans "WARNING: The following services depend on this volume and could be disrupted when the volume is detached: " %}
```
Could "this volume" be reworked to show the name of the volume?